### PR TITLE
ADDED package level comments for: benchmark/, examples/helloworld/

### PR DESCRIPTION
**Pull request by zelahi**
Description: In this pull request, we are adding comments at the package level for multiple files. I generated these comments based on the documentation here: https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Ref: #1967

### CODE REVIEW

**Comment by zelahi** [comment]
I signed it!

**Comment by zelahi** [comment]
Is there a way to have travis-ci re-test?

**Comment by dfawley** [comment]
If you push new changes, travis will re-run.  Re-running the same change won't help in this case -- from the output:

```
+gofmt -s -d -l .
+fail_on_output
+tee /dev/stderr
+read
benchmark/client/main.go
```

That means `benchmark/client/main.go` isn't `gofmt`'d correctly.  Please reformat and try again.  FYI, you can run `vet.sh` locally so you don't have to iterate on travis.  (The output can be admittedly hard to parse sometimes, though.)

**Comment by zelahi** [comment]
aww tho it looks like `vet.sh` didn't work on my Mac

File: benchmark/client/main.go [hunk 1]
```diff
@@ -16,6 +16,27 @@
...
 
+/*
+Package main provides a client used for benchmarking.  Before running the
+client, the user would need to launch the grpc server before running the
+client.
+
+To start the server before running the client, you can run something similar
+to the command below:
+
+go run benchmark/server/main.go -test_name=grpc_test
```
Comment (canguler) [inline benchmark/client/main.go]: Instead of duplicating this command here, can you provide a pointer to the other file?

File: benchmark/client/main.go [hunk 2]
```diff
@@ -16,6 +16,27 @@
  *
  */
 
+/*
+Package main provides a client used for benchmarking.  Before running the
+client, the user would need to launch the grpc server before running the
```
Comment (canguler) [inline benchmark/client/main.go]:
```suggestion
client, the user would need to launch the grpc server.
```

File: benchmark/client/main.go [hunk 3]
```diff
@@ -16,6 +16,27 @@
...
+client.
```
Comment (canguler) [inline benchmark/client/main.go]:
```suggestion
```

File: benchmark/client/main.go [hunk 4]
```diff
@@ -16,6 +16,27 @@
...
+
+To start the server before running the client, you can run something similar
+to the command below:
+
+go run benchmark/server/main.go -test_name=grpc_test
+
+After starting the server, the client can be ran.  An example of how to run this
```
Comment (canguler) [inline benchmark/client/main.go]:
```suggestion
After starting the server, the client can be run.  An example of how to run this
```

File: benchmark/client/main.go [hunk 5]
```diff
@@ -16,6 +16,27 @@
...
+command is:
+
+go run benchmark/client/main.go -test_name=grpc_test
+
+If the server is running on a different port than 50051, then use the port flag
+for the client to hit the server on the correct port.
+An example for how to run this command which a different port is:
```
Comment (canguler) [inline benchmark/client/main.go]:
```suggestion
An example for how to run this command on a different port is:
```

File: benchmark/server/main.go [hunk 6]
```diff
@@ -16,6 +16,16 @@
...
  */
 
+/*
+Package main provides a server used for benchmarking.  It launches a server
+which is listening on port 50051. An example to start the server would be to
+use:
+
+go run benchmark/server/main.go -test_name=grpc_test
+
+After starting the server, the client can be ran separately and used to test
```
Comment (canguler) [inline benchmark/server/main.go]:
```suggestion
After starting the server, the client can be run separately and used to test
```

File: benchmark/stats/stats.go [hunk 7]
```diff
@@ -16,6 +16,7 @@
  *
  */
 
+//Package stats registers stats used for creating benchmarks
```
Comment (canguler) [inline benchmark/stats/stats.go]:
```suggestion
// Package stats registers stats used for creating benchmarks
```

File: benchmark/stats/util.go [hunk 8]
```diff
@@ -16,6 +16,8 @@
  *
  */
 
+// Package stats registers benchmark statistics. utils.go provide methods to
+// add statistics for benchmarks
```
Comment (canguler) [inline benchmark/stats/util.go]: Package level comments are generally put into a single file. So, this one isn't necessary.

File: examples/helloworld/greeter_server/main.go [hunk 9]
```diff
@@ -18,6 +18,7 @@
 
 //go:generate protoc -I ../helloworld --go_out=plugins=grpc:../helloworld ../helloworld/helloworld.proto
 
+// greeter_server command ...
```
Comment (canguler) [inline examples/helloworld/greeter_server/main.go]: As this is a package level comment, it should start with "Package main ..."
```suggestion
// Package main implements a server for Greeter service. 
```

File: examples/helloworld/greeter_client/main.go [hunk 10]
```diff
@@ -16,6 +16,7 @@
  *
  */
 
+// greeter_client command ...
```
Comment (canguler) [inline examples/helloworld/greeter_client/main.go]: As this is a package level comment, it should start with "Package main ..."
```suggestion
// Package main implements a client for Greeter service.
```

File: benchmark/server/main.go [hunk 11]
```diff
@@ -16,6 +16,16 @@
  *
  */
 
+/*
+Package main provides a server used for benchmarking.  It launches a server
+which is listening on port 50051. An example to start the server would be to
```
Comment (canguler) [inline benchmark/server/main.go]: You have double space after the sentences for the rest of the PR.
```suggestion
which is listening on port 50051.  An example to start the server would be to
```

**Comment by zelahi** [comment]
Thank you for the feedback.  I'll go ahead and make the requested changes

File: benchmark/client/main.go (hunk 1 above)
Comment (canguler) [reply benchmark/client/main.go]: @zelahi can you address this one as well?

Comment (zelahi) [reply benchmark/client/main.go]: apologies, I'll be addressing this in another commit

File: benchmark/server/main.go [hunk 12]
```diff
@@ -16,6 +16,15 @@
  *
  */
 
+/*
+Package main provides a server used for benchmarking.  It launches a server
+which is listening on port 50051.  An example to start the server can be found
+at:
+	benchmark/server/main.go
```
Comment (canguler) [inline benchmark/server/main.go]: This is the mentioned file. Please provide the example on how to run the server here.
